src/twisted/trial/test/detests.py

```diff
@@ -186,6 +186,24 @@ def test_timeoutZero(self):
 
     test_timeoutZero.timeout = 0  # type: ignore[attr-defined]
 
+    def test_addCleanupPassDefault(self):
+        # test default timeout for addCleanup functions
+        def cleanup():
+            d = defer.Deferred()
+            reactor.callLater(0, d.callback, "success")
+            return d
+
+        self.addCleanup(cleanup)
+
+    def test_addCleanupTimeout(self):
+        # test that functions added via addCleanup supports timeout
```

Comment:
Can we have this as a docstring ?

What does "support timeout" mean ?

Maybe something like this

```suggestion
        """
        When the deferred returned by addCleanup is not
        resolved sooner than the test's timeout,
        the test is considered failed.
        
        For this test, we retrun from C{addCleanup} a
        deferred that will never receive a result.
```

Reply:
Fixed.